plugins/linux-pipewire/pipewire.c

```diff
@@ -143,6 +143,7 @@ struct _obs_pipewire_stream {
 		int release_syncobj_fd;
 		uint64_t acquire_point;
 		uint64_t release_point;
+		bool primed_release_point;
```

Comment:
`signaled_release_point` to keep the terminology consistent.

Reply:
It's not exactly "signaled" because it's also set when it's scheduled to signal at some point in the future (at the end of the normal video rendering path)

Reply:
If you really want to call it SubmitOrSignaled im ok with that too.

Reply:
Done